ODT/ODTmeta.php

```diff
@@ -17,13 +17,16 @@ class ODTMeta
      * Constructor. Set initial meta data.
      */
     public function __construct() {
+        global $conf;
+        $dateTime = new DateTime();
+        
         $this->meta = array(
                 'meta:generator'            => 'DokuWiki '.getversion(),
                 'meta:initial-creator'      => 'Generated',
-                'meta:creation-date'        => date('Y-m-d\\TH::i:s', null), //FIXME
+                'meta:creation-date'        => $dateTime->format("Y-m-d\\TH::i:s"),
                 'dc:creator'                => 'Generated',
-                'dc:date'                   => date('Y-m-d\\TH::i:s', null),
-                'dc:language'               => 'en-US',
+                'dc:date'                   => $dateTime->format("Y-m-d\\TH::i:s"),
+                'dc:language'               => $conf['lang'],
```

Comment:
So far I know `$conf['lang']` has format like `'en'`, and not `'en-US'`. Are you sure the ODT format can handle also the 'en' format, thus without country part?
A quick search gives the impression that the country part is needed... https://github.com/jgm/pandoc/issues/1667#issuecomment-310853901

Reply:
Hello @Klap-in, thank you for your response. You're right. I tried to find a way to properly detect the language and country code but I was unable to, so I fallback to en-US again.